references.go

```diff
@@ -39,26 +42,56 @@ func (w *Workspace) locateReferences(obj types.Object, pkg *Package) []*ref {
 			})
 		}
 	}
+	fmt.Printf("Starting with %d local refs\n", len(refs))
 
 	if obj.Exported() {
+		fmt.Printf("Exported obj\n")
 		n, ok := w.Loader.caravan.Find(pkg.absPath)
 		if !ok {
 			// Should never get here.
 			panic("Shit.")
 		}
 		fmt.Printf("obj: %#v\n", obj)
 		asc := flattenAscendants(n)
-		refs = checkAscendants(asc, obj)
+		fmt.Printf("Have %d ascendants\n", len(asc))
+		ascRefs := w.checkAscendants(asc, obj)
+		for _, r := range ascRefs {
+			refs = append(refs, r)
+		}
 	}
 
+	fmt.Printf("Returning %d refs\n", len(refs))
 	return refs
 }
 
-func checkAscendants(ascendants map[string]*Package, obj types.Object) []*ref {
+func (w *Workspace) checkAscendants(ascendants map[string]*Package, obj types.Object) []*ref {
```

Comment:
Once debugging messages are removed, this function will no longer need `*Workspace`